plugins/MauticOutlookBundle/README.md

```diff
@@ -0,0 +1,13 @@
+# Outlook Plugin for Mautic
```

Comment:
Let's remove this from here and make sure it's added to the documentation repo if not already.